### ISSUE TITLE
Custom topbar title does not take the whole space and looks shifted (iOS + Android)

### ISSUE THREAD

**Issue by samcampisi** [wix/react-native-navigation#6456]
### Issue Description

I've seen some issues related to the ```alignment``` value for the title in the topbar options, but I'm having trouble with this.

I'm trying to render a custom title component in a topbar with "alignment fill", but the custom component does not take the whole space.

Here's how it looks on both platforms: 

iOS:
<img width="400" alt="Screenshot 2020-08-05 at 13 53 28" src="https://user-images.githubusercontent.com/42766924/89409869-0b33e900-d723-11ea-8d03-c07865bb9aa9.png">

Android:
<img width="554" alt="Screenshot 2020-08-05 at 13 55 12" src="https://user-images.githubusercontent.com/42766924/89410027-49c9a380-d723-11ea-8039-85d64a904fb7.png">

### Steps to Reproduce / Code Snippets / Screenshots

I looked to see if I had some weird style but after reducing it to the minimum I still observe this.

Here are the topbar options:
```
topBar: {
        visible: true,
        title: {
          component: {
            name: 'app.CustomTopBar',
            aligment: 'fill',
          },
          aligment: 'fill',
        },
        backButton: {
          visible: false,
        },
},
```

And the view in the custom component (aka app.CustomTopBar)
```
<View style={{
    flex: 1,
    margin: 0,
    padding: 0,
    height: 80,
    flexDirection: 'row',
    justifyContent: 'space-between',
    backgroundColor: 'pink'
}}>
        <Text>A</Text>
        <Text>B</Text>
        <Text>C</Text>
</View>
```

Where does this left shifting come from? Is there anything I could do to make this occupy the whole space? 
Even after giving it width: '100%' or width the size of the screen this space is still there

---
### Environment
* React Native Navigation version: 6.11
* React Native version: 0.62.2
* Platform(s) (iOS, Android, or both?): Both
* Device info (Simulator/Device? OS version? Debug/Release?): iOS 13.6 in an iPhoneSE Simulator, Android 9 Xiaomi Mi A1

**Comment by jinshin1013** [comment]
Hi @samcampisi, thanks for reporting the issue! It'd be great if you could fork the project and push a reproduction in the Playground app so that we could have more close look at the issue. Cheers!

**Comment by guyca** [comment]
Hey @samcampisi 
Seems like there's a typo in the options declaration. It should be `alignment`.
Also, if you want to set a background component which will stretch across the entire TopBar, then use:
```js
topBar: {
  background: {
    component: {
      name: 'MyBackGroundComponent'
    }
  }
}
```

**Comment by samcampisi** [comment]
@guyca yes you're right 🤦 the things you don't see after looking at it for hours...

Anyway, after setting ```alignment``` properly I still see the space like this:

iOS:
<img width="402" alt="Screenshot 2020-08-06 at 08 30 54" src="https://user-images.githubusercontent.com/42766924/89498749-25bba000-d7bf-11ea-985a-07fe273731b4.png">

Android:
<img width="560" alt="Screenshot 2020-08-06 at 08 31 11" src="https://user-images.githubusercontent.com/42766924/89498770-30763500-d7bf-11ea-86ff-f8354baeaed3.png">

I'm gonna try pushing a reproduction in the playground and thank you so much for taking a look at it!

**Comment by guyca** [comment]
Hey @samcampisi This is the expected behaviour, the title view doesn't stretch across the entire width of the TopBar.
I'm closing the issue as we looked into this a while ago and there's nothing much we can do about this.

**Comment by samcampisi** [comment]
Oh I see. Thanks anyway! I'm gonna try to fix the background issue in my code since it does work in the playground app.

**Comment by zabojad** [comment]
@guyca it's not really a solution as the background component doesn't seem to support Touchables...

**Comment by swornov** [comment]
i change my phone around that time

**Comment by swornov** [comment]
you think you can fix this?

**Comment by yedidyak** [comment]
Closing this as part of a maintainer cleanup of open issues older than one year.

Decision: Closed: non-critical feature request/question/discussion.

If this still reproduces on the latest React Native Navigation with a minimal reproduction, please open a new issue with current RN/RNN versions and repro steps.
